.github/workflows/failpoint_test.yaml

```diff
@@ -15,5 +15,6 @@ jobs:
         with:
           go-version: ${{ steps.goversion.outputs.goversion }}
       - run: |
+          sudo make root-test
```

Comment:
Suggest to add a separate workflow test for the data corruption test, i.e. `data_corruption_test.yaml`. Eventually we will setup nightly test for it.

We can address this in a followup PR along with https://github.com/etcd-io/bbolt/pull/622